firmware/temperature_detection/user_config.h

```diff
@@ -1,12 +1,16 @@
 #ifndef USER_CONFIG_H
 #define USER_CONFIG_H
 
+// Switches
+#define ENABLE_SONAR
```

Comment:
@1technophile, if you prefer to choose another name or wrap the switch with some code to detect M5StickC, feel free to push a new commit to this pull request.

Reply:
I like it this way, we will use a `build_flag `on PIO to override the instruction on `user_config.h`

Reply:
:shipit: then?